packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneBlobs.spec.ts

```diff
@@ -0,0 +1,192 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import {
+    IGCRuntimeOptions,
+    ISummarizer,
+} from "@fluidframework/container-runtime";
+import {
+    requestFluidObject } from "@fluidframework/runtime-utils";
+import {
+    ITestObjectProvider,
+    createSummarizerWithContainer,
+    summarizeNow,
+    waitForContainerConnection,
+    mockConfigProvider,
+    ITestContainerConfig,
+} from "@fluidframework/test-utils";
+import { describeNoCompat, ITestDataObject, itExpects } from "@fluidframework/test-version-utils";
+import { delay, stringToBuffer } from "@fluidframework/common-utils";
+import { LoaderHeader } from "@fluidframework/container-definitions";
+
+describeNoCompat("GC tombstone blob tests", (getTestObjectProvider) => {
```

Comment:
There tests won't be enough. Attachment blobs have different behavior based on whether they were loaded during detached or attached state. Also, blobs can be de-duped which adds to the mix.
See https://github.com/microsoft/FluidFramework/blob/0f256a1be3e0a60d51729ea7fd3bf1232ef2a073/packages/test/test-end-to-end-tests/src/test/blobs.spec.ts#L485 for these tests. Maybe just run the same scenarios with tombstones enabled.